### PR TITLE
[FLAKY TESTS] Update and clean up DilatedTests

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: src/core/Akka.TestKit.Tests/TestKitBaseTests/DilatedTests.cs [hunk 1]
```diff
@@ -26,47 +28,49 @@ public DilatedTests()
...
         }
 
         [Fact]
         public void AwaitCondition_should_dilate_timeout()
         {
-            var before = Now;
-            Intercept(() => AwaitCondition(() => false, TimeSpan.FromMilliseconds(Timeout)));
-            var after = Now;
-            var diff = (after - before).TotalMilliseconds;
-            Assert.True(Math.Abs(diff - ExpectedTimeout) <= DiffDelta);
```
Comment (Arkatufus) [inline src/core/Akka.TestKit.Tests/TestKitBaseTests/DilatedTests.cs:39]: Timeout should always be greater than the dilated timeout time, not plus or minus a certain margin.

File: src/core/Akka.TestKit.Tests/TestKitBaseTests/DilatedTests.cs [hunk 2]
```diff
@@ -6,17 +6,19 @@
...
 using Xunit;
+using Xunit.Sdk;
 
 namespace Akka.Testkit.Tests.TestKitBaseTests
 {
     public class DilatedTests : AkkaSpec
     {
         private const int TimeFactor = 4;
-        private const int Timeout = 50;
+        private const int Timeout = 1000;
```
Comment (Arkatufus) [inline src/core/Akka.TestKit.Tests/TestKitBaseTests/DilatedTests.cs:19]: Need a value big enough that other timing noise (such as GC) can be excluded from the result

**Comment by Aaronontheweb** [comment]
@Arkatufus a bunch of these all failed

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Comment by Aaronontheweb** [comment]
@Arkatufus looks like another racy test in this suite Akka.TestKit.Tests.Xunit2.TestEventListenerTests.CustomEventFilterDebugTests.Make_sure_async_works

Unrelated to your changes.

**Pull request merged**